src/components/todo-item/TodoItem.jsx

```diff
@@ -0,0 +1,19 @@
+import style from "./TodoItem.module.css";
+
+export function TodoItem(props) {
+  const {
+    todo: { id, content, checked },
+  } = props;
+  return (
+    <div className={style.item}>
+      <div className={style.content}>
+        <input type="checkbox" checked={checked} />
+        {content}
+      </div>
+      <div>
+        <button className="edit-btn">Edit</button>
+        <button className="delete-btn">Delete</button>
```

Comment:
Why do we need to add a class name to these btns if we don't have any styles for them?